main.nf

```diff
@@ -675,7 +815,7 @@ if( params.aligner =~ /bismark/ ){
         if(params.single_end) {
```

Comment:
Why did you remove the cytosine report parameter?

Reply:
Again, didn't see it exists in the newer versions. I didn't touch this step myself... Fixed